src/io/flutter/editor/WidgetIndentsHighlightingPassFactory.java

```diff
@@ -295,6 +168,8 @@ void runWidgetIndentsPass(EditorEx editor, @NotNull FlutterOutline outline) {
       return;
     }
 
+    // QUESTION: why doesn't this check against isShowBuildMethodGuides ?
```

Comment:
Now that you have refactored, I agree it is safest to check against the flag here.
It is possible it wasn't needed before as this wouldn't trigger unless the guides were enabled.